app/views/pages/index.html.haml

```diff
@@ -57,7 +57,7 @@
             = graphical_icon :exercises, hex: true
             %h2.text-h2
               Over
-              %strong 3,450 coding exercises.
+              %strong #{Exercise.available.count} coding exercises.
```

Comment:
I presume this is lightening fast? (I'm protective over home page performance 😉)

Reply:
This is fine :)